server/tests/transaction/service/test_payment.py

```diff
@@ -126,24 +132,42 @@ async def test_existing_transaction(
 
         assert transaction.id == existing_transaction.id
 
+    @pytest.mark.parametrize(
+        "risk",
+        [
+            pytest.param({"risk_level": None, "risk_score": None}),
+            pytest.param({"risk_level": "normal", "risk_score": 20}),
+        ],
+    )
     async def test_customer_user(
         self,
         session: AsyncSession,
         save_fixture: SaveFixture,
         pledge: Pledge,
         user: User,
         stripe_service_mock: MagicMock,
+        risk: dict[str, Any],
     ) -> None:
         user.stripe_customer_id = "STRIPE_CUSTOMER_ID"
         await save_fixture(user)
         pledge.payment_id = "STRIPE_PAYMENT_ID"
         await save_fixture(pledge)
 
+        risk_level = risk["risk_level"]
+        if risk_level:
+            risk_level = str(risk_level)
+
+        risk_score = risk["risk_score"]
+        if risk_score is not None:
+            risk_score = int(risk_score)
+
```

Comment:
To avoid all the pain with type checker, declare parameters separately in the pytest parametrization:

```py
@pytest.mark.parametrize(
    "risk_level,risk_score",
    [
        pytest.param(None, None),
        pytest.param("normal", 20),
    ],
)
async def test_customer_user(
    self,
    session: AsyncSession,
    save_fixture: SaveFixture,
    pledge: Pledge,
    user: User,
    stripe_service_mock: MagicMock,
    risk_level: str | None,
    risk_score: int | None,
) -> None:
    ...
```

Reply:
Nice - thank you!